src/SprykerSdk/SyncApi/OpenApi/Updater/OpenApiUpdater.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\SyncApi\OpenApi\Updater;
+
+use ArrayObject;
+use cebe\openapi\Reader;
+use cebe\openapi\spec\OpenApi;
+use cebe\openapi\Writer;
+use SprykerSdk\SyncApi\Message\MessageBuilderInterface;
+use SprykerSdk\SyncApi\OpenApi\Builder\FilepathBuilderInterface;
+use SprykerSdk\SyncApi\SyncApiConfig;
+use Throwable;
+use Transfer\OpenApiResponseTransfer;
+use Transfer\UpdateOpenApiRequestTransfer;
+
+class OpenApiUpdater implements OpenApiUpdaterInterface
+{
+    /**
+     * @var \SprykerSdk\SyncApi\Message\MessageBuilderInterface
+     */
+    protected $messageBuilder;
```

Comment:
```suggestion
    protected MessageBuilderInterface $messageBuilder;
```

Can you update all properties to have this format please?